packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -194,16 +218,61 @@ const Autocomplete = <
     [errorMessage, hint, isOptional, label, nameOverride]
   );
 
+  const defaultValuesProp:
+    | AutocompleteValue<
+        OptionType,
+        HasMultipleChoices,
+        undefined,
+        IsCustomValueAllowed
+      >
+    | undefined = useMemo(() => {
+    if (hasMultipleChoices) {
+      return defaultValue == undefined
+        ? ([] as AutocompleteValue<
+            OptionType,
+            HasMultipleChoices,
+            undefined,
+            IsCustomValueAllowed
+          >)
+        : defaultValue;
+    }
```

Comment:
I don't like using `==` over the more explicit `===` because I can't tell from looking at the code if we care about `null` or `false` or `0`. I wanna know your intent.

Reply:
Yes it should be `===`... this was a typo